.github/workflows/build_and_test.yml

```diff
@@ -350,11 +351,12 @@ jobs:
           real_s3_secret_access_key: "${{ secrets.AWS_SECRET_ACCESS_KEY_CI_TESTS_S3 }}"
           rerun_flaky: true
         env:
+          DEFAULT_PG_VERSION: ${{ matrix.pg_version }}
           TEST_RESULT_CONNSTR: ${{ secrets.REGRESS_TEST_RESULT_CONNSTR }}
           CHECK_ONDISK_DATA_COMPATIBILITY: nonempty
 
       - name: Merge and upload coverage data
-        if: matrix.build_type == 'debug'
+        if: matrix.build_type == 'debug' && matrix.pg_version == 'v14'
```

Comment:
why for only one version?

Reply:
It's a good question.
[neondatabase/zenith-coverage-data](https://github.com/neondatabase/zenith-coverage-data) repo is huge, so I didn't want to put twice more data per run into it.

Some time ago we had a meeting with @shanyp regarding making coverage move useful (i.e. posing summary to PRs and so on), I think we can revisit this after (or within) https://github.com/neondatabase/neon/issues/3543.

Doesn't look like a blocker for the PR.